src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -20711,9 +20741,12 @@ public static bool TryGetValue([MaybeNullWhen(true)] out string s)
     }
 }
 ";
-            // Since we can't analyze this well enough, we loosen the type unconditionally
             var comp = CreateNullableCompilation(new[] { source, MaybeNullWhenAttributeDefinition });
-            comp.VerifyDiagnostics();
+            comp.VerifyDiagnostics(
+                // (11,13): error CS8762: Parameter 's' may not have a null value when exiting with 'false'.
```

Comment:
>error [](start = 28, length = 5)

I expected this to be a warning. Did I misinterpret something?

Reply:
Not sure how I messed up the comment. I"ll fix